npm/templates/deploy.py

```diff
@@ -89,13 +82,9 @@
     os.path.realpath('external/nodejs_windows_amd64/bin/'),
 ])
 
-with open(expect_input_file.name) as expect_input:
-    subprocess.check_call([
-        '/usr/bin/expect',
-    ], stdin=expect_input, env={
-        'PATH': node_path,
-        'PASSWORD': npm_password
-    })
+f = open(".npmrc", "w")
+f.write("//%s:_authToken=%s" % (npm_registry, npm_token))
+f.close()
```

Comment:
Previously we relied on `npm adduser` to generate this `.npmrc` file by signing in to the remote npm repo. Now, we create it ourselves manually based on the remote repo name and the provided auth token.